Components/Combat/TrinityCombat.cs

```diff
@@ -70,9 +79,14 @@ public static async Task<bool> MainCombatTask()
             if (Routines.Current.ShouldReturnStartResult)
                 return startHookResult;
 
-            await UsePotion.Execute();
+            if (!await UsePotion.DrinkPotion())

```

Comment:
this should return true when we actually drink a potion? None of the below code will be running when we don't need to drink any potions?